agent/Agent.js

```diff
@@ -210,18 +210,18 @@ class Agent extends EventEmitter {
       console.warn('unable to get the node for scrolling');
       return;
     }
-    var element = node.nodeType === Node.ELEMENT_NODE ?
+    var internalInstance = node.nodeType === Node.ELEMENT_NODE ?
```

Comment:
This one is actually right.. somewhat. It refers to a DOM node. Let's call it `domElement`